packages/fast-web-utilities/src/localization.ts

```diff
@@ -10,7 +10,7 @@ export const localeDirectionMapping: { [key: string]: Direction } = {
     en: Direction.ltr,
 };
 
-export default function isRTL(locale: string): boolean {
```

Comment:
are we using this anywhere? I'm not sure how useful this really is with this implementation, because `isRTL` doesn't actually tell us anything about the environment we're in - which is usually what we want to know

Reply:
no clue - @janechu can you advise on this?

Reply:
We were using it in the dev-site way back, but I'm not sure its being used anywhere since then.

Reply:
@janechu @nicholasrice that's the only place I can find us using it; should we just move it in there only?

Reply:
I've moved this out